shadow-dom/declarative/gethtml.html

```diff
@@ -47,12 +46,7 @@
           `shadowrootmode=${mode}${delegatesAttr}${serializableAttr}` +
           `${clonableAttr}>`;
       wrapper.setHTMLUnsafe(html);
-      if (isOpen) {
-        shadowRoot = wrapper.firstElementChild.shadowRoot;
-      } else {
-        // For closed shadow root, we rely on the behavior of attachShadow to return it to us
```

Comment:
It'd be nice to keep this comment, so we know why we're not just grabbing `child.shadowRoot`.